index.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const cors = require('cors');
+const { pool } = require('./config');
+
+const app = express();
+
+app.use(express.json());
+app.use(express.urlencoded());
+app.use(cors());
+
+const getTrees = (request, response) => {
+  pool.query('SELECT * FROM trees', (error, results) => {
+    if (error) {
+      throw error
+    }
+    response.status(200).json(results.rows)
+  })
+  console.log('get trees request')
+}
+
+  const getZip = (request, response) => {
+    pool.query('SELECT * FROM zip_codes', (error, results) => {
+      if (error) {
```

Comment:
One small note I might make here is that if there is an error, it would be helpful to send back a response to the FE with more context.  Throwing an error will break things...and while our FE may get a 500 error automatically, we could send back the error message using something like `response.status(500).json({ error });`.